test/integration/api-schema/basic.js

```diff
@@ -24,50 +24,74 @@ const next = afterAll(function (err, validators) {
   test('metadata schema failure', function (t) {
     t.strictEqual(validateMetadata({}), false)
     validateFieldMessages(t, validateMetadata.errors, [
-      { field: 'data.service', message: 'is required' }
+      { message: 'should have required property \'service\'' }
     ])
     t.end()
   })
 
   test('transaction schema failure', function (t) {
     t.strictEqual(validateTransaction({}), false)
     validateFieldMessages(t, validateTransaction.errors, [
-      { field: 'data.duration', message: 'is required' },
-      { field: 'data.type', message: 'is required' },
-      { field: 'data.id', message: 'is required' },
-      { field: 'data.trace_id', message: 'is required' },
-      { field: 'data.span_count', message: 'is required' }
+      { message: 'should have required property \'duration\'' },
+      { message: 'should have required property \'id\'' },
+      { message: 'should have required property \'span_count\'' },
+      { message: 'should have required property \'trace_id\'' },
+      { message: 'should have required property \'type\'' }
     ])
     t.end()
   })
 
   test('span schema failure', function (t) {
     t.strictEqual(validateSpan({}), false)
     validateFieldMessages(t, validateSpan.errors, [
-      { field: 'data.duration', message: 'is required' },
-      { field: 'data.name', message: 'is required' },
-      { field: 'data.type', message: 'is required' },
-      { field: 'data.id', message: 'is required' },
-      { field: 'data.trace_id', message: 'is required' },
-      { field: 'data.parent_id', message: 'is required' },
-      { field: 'data', message: 'no schemas match' }
+      { message: 'should have required property \'duration\'' },
+      { message: 'should have required property \'id\'' },
+      { message: 'should have required property \'name\'' },
+      { message: 'should have required property \'parent_id\'' },
+      { message: 'should have required property \'trace_id\'' },
+      { message: 'should have required property \'type\'' },
+      { message: 'should have required property \'start\'' },
+      { message: 'should have required property \'timestamp\'' },
+      { message: 'should match some schema in anyOf' }
     ])
     t.end()
   })
 
   test('error schema failure', function (t) {
     t.strictEqual(validateError({}), false)
     validateFieldMessages(t, validateError.errors, [
-      { field: 'data', message: 'no schemas match' },
-      { field: 'data.id', message: 'is required' }
+      { message: 'should have required property \'id\'' },
+      { message: 'should have required property \'exception\'' },
+      { message: 'should have required property \'log\'' },
+      { message: 'should match some schema in anyOf' }
     ])
```

Comment:
The `exception`, `log` and `anyOf` messaged in our new code don't have equivalents in the old.  Again, presuming this is deliberate and just making note of it out loud.